samples/websocket/chatApp/bundle/resources/pom.xml

```diff
@@ -1,451 +0,0 @@
-<!--
```

Comment:
Why are we removing this?

Reply:
This removes the sample distribution used to deploy WebSocket bundle sample since that can be deployed in test-distribution.